geode-core/src/main/java/org/apache/geode/internal/cache/backup/BackupTask.java

```diff
@@ -192,7 +192,7 @@ boolean isCancelled() {
     return isCancelled;
   }
 
-  void waitForBackup() {
+  void waitTillBackupFilesAreCopiedToTemporaryLocation() {
```

Comment:
The name suggests too much of the internal implementation. Maybe something like: "waitUntilDestroysAllowed" or "waitForFileCopyCompletion", something like that?

Reply:
It was done only on BackupTask...The reason was to help the reader to know when the wait will terminate.